### ISSUE TITLE
Should there be (a11y) commit type? How do you address accessibility UI related changes?

### ISSUE THREAD

**Issue by webmatrixxxl** [conventional-commits/conventionalcommits.org#577]
Hello!

In the Front-End world recently because of the European Accessibility Act, we have been required (for good) to implement a lot of accessibility across components and pages

And I started wondering how we should address accessibility in the commit messages.

`
a11y(checkout): add alt text to card types in checkout
`
or

`
accessibility(checkout): add alt text to card types in checkout
`

or we can stick to

`
feat(checkout): add alt text to card types in checkout
`

or maybe

`
chore(checkout): add alt text to card types in checkout
`

Personally, I kind of prefer the first one 

`
a11y(checkout): add alt text to card types in checkout
`

What do you think?

**Comment by henrikhelmers** [comment]
On the one hand, accessibility should be part of code quality (be it by adding alt text or correctly implementing design), which would make "chore" correct.

On the other hand it would be valuable to be able to point at commits related to accessibility to document effort in that area. I also belive that as the amount of awareness (and legislation) increase, the amount of commits directly related to accessibility will as well.

**Comment by gekoke** [comment]
`chore(a11y):`
